### PR TITLE
[FIX] project,sale_project: fix action of milestone stat button

### PR DESCRIPTION
**Pull request by kcv-odoo**
This commit fix button action of milestone stat button.

task-2784776

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/models/project_milestone.py [hunk 1]
```diff
@@ -53,21 +53,3 @@ def _get_data(self):
...
-            'domain': "[('project_id', '=', active_id)]",
-            'context': {**self.env.context, 'default_project_id': self.env.context.get('active_id')},
-            'help': """
-                <p class="o_view_nocontent_smiling_face">
-                No milestones found. Let's create one!
-                </p><p>
-                Track major progress points that must be reached to achieve success.
-                </p>
-            """,
-        }
```
Comment (xavierbol) [inline addons/project/models/project_milestone.py:73]: You cannot remove a method in stable, you need to deprecate it. 
```suggestion
    def action_get_list_view(self):
        # TODO: Remove me in master
        return {}
```

File: addons/sale_project/models/project_milestone.py [hunk 2]
```diff
@@ -19,10 +19,3 @@ class ProjectMilestone(models.Model):
     @api.model
     def _get_fields_to_export(self):
         return super()._get_fields_to_export() + ['allow_billable', 'quantity_percentage', 'sale_line_name']
-
-    def action_get_list_view(self):
-        view = super().action_get_list_view()
-        project = self.env['project.project'].browse(self.env.context.get('active_id'))
-        if project.allow_billable:
-            view['views'] = [(self.env.ref('sale_project.project_milestone_view_tree').id, 'tree'), (False, 'form')]
-        return view
```
Comment (xavierbol) [inline addons/sale_project/models/project_milestone.py:28]: You cannot remove the method you should deprecate it in stable and then in master you can remove it.

**Review by xavierbol — COMMENTED** [review]
Thanks for the fix I have made 2 comments and after that it will look good to me.

When the fix will be ready to be deployed in master, can you create the action in xml instead? Thanks in advance. 🙂

File: addons/sale_project/models/project_milestone.py (hunk 2 above)
Comment (IT-Ideas) [reply addons/sale_project/models/project_milestone.py:28]: As the parent method is deprecated we should be safe. It looks good to me. Thanks!

**Comment by IT-Ideas** [comment]
Hello @kcv-odoo as @aju-odoo requested this PR to be urgently merged, it will skip the stages and I will test it myself.

**Review by IT-Ideas — APPROVED** [review]
Thanks for your fix 🙂 
@robodoo r+

**Pull request closed without merging**